src/app/layout.tsx

```diff
@@ -39,7 +39,7 @@ export default async function RootLayout({
         </Script>
       </head>
 
-      <body>
+      <body suppressHydrationWarning>
```

Comment:
What does it do Seb?

Reply:
That's to avoid extension blocking the hydratation because they are adding extra stuff to the body element (like grammarly, etc..)